src/app/tests/suites/TestDiscovery.yaml

```diff
@@ -146,7 +149,7 @@ tests:
       arguments:
           values:
               - name: "value"
-                value: discriminator
+                value: shortDiscriminator
```

Comment:
@vivien-apple I did this change and updated DiscoveryCommands to not do `>> 8) & 0x0F` on the argument as it seems clearer to specify what short discriminator is actually meant.

Not sure if this has other side effects though. I can also match python to assume that the value for the "find by short discriminator" is the "long discriminiator". Thoughts?

Reply:
LGTM. I can't remember why I did it this way. I think this was because it was a shorter to have a single discriminator to update when passing argument and not have to calculate the second one.
But it that is unclear, that is OK.